### ISSUE TITLE
'/body.1/Unsqueeze_output_0'

### ISSUE THREAD

**Issue by NenelFX** [chaiNNer-org/chaiNNer#2927]
An error occurred in a Convert To NCNN node:

'/body.1/Unsqueeze_output_0'

Input values:
• ONNX Model: Value of type 'nodes.impl.onnx.model.OnnxGeneric'
• Data Type: fp16

```py
Traceback (most recent call last):
  File "D:\Video_Tools\chaiNNer\resources\src\process.py", line 176, in run_node
    raw_output = node.run(*enforced_inputs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Video_Tools\chaiNNer\resources\src\packages\chaiNNer_onnx\onnx\utility\convert_to_ncnn.py", line 41, in convert_to_ncnn_node
    ncnn_model = NcnnModelWrapper(converter.convert(fp16, False))
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Video_Tools\chaiNNer\resources\src\nodes\impl\onnx\onnx_to_ncnn.py", line 3507, in convert
    slope = self.weights[node.input[1]]
            ~~~~~~~~~~~~^^^^^^^^^^^^^^^
KeyError: '/body.1/Unsqueeze_output_0'
```

model: 2x_AniSD_PS_G6i2_Compact_50K_fp32.onnx

Windows 10
GPU 920m 1gb

**Comment by RunDevelopment** [comment]
@joeyballentine Didn't 2x compact models use an operation NCNN doesn't support?

**Comment by joeyballentine** [comment]
I think that was the 2x realesrgan models using pixelunshuffle. This might be a similar thing though. Most likely this is not something we can fix

**Comment by NenelFX** [comment]
I managed to convert some of the ONNX models from "https://github.com/Sirosky/Upscale-Hub/releases/tag/AniSD".
other models have the same error
and others were converting, I waited a long time but they didn't convert.
the models that worked they converted almost instantly.
I also tried other onnx models from others and augms convert others show the same error.
note: my gpu is nvidia but it is old and models that use cuda do not work only ncnn works
and my cpu is an intel n3700 it does not support avx2
